googleapiclient/discovery.py

```diff
@@ -17,18 +17,19 @@
 A client library for Google's discovery based APIs.
 """
 from __future__ import absolute_import
-import six
 
 __author__ = "jcgregorio@google.com (Joe Gregorio)"
 __all__ = ["build", "build_from_document", "fix_method_name", "key2param"]
 
-from six.moves import http_client
-from six.moves.urllib.parse import urljoin  
+import http.client as http_client
```

Comment:
Nit:  we'd generally prefer to leave the "normal" import in place.  Since this is a "cleanup" PR, I suggest:
```suggestion
import http.client
```
and then replace the mentions below with `http.client`.

Reply:
I agree with this comment but I'm unable to make the change due to existing variables named `http`. For example, https://github.com/googleapis/google-api-python-client/blob/master/googleapiclient/http.py#L901 .

How do you feel about leaving this as is?

Reply:
OK, given that the `http.client` module is actually used in functions which also take `http` as an argument (and that those functions are part of the library's public API), leaving the import as you have it is the best option.